src/subcommands/server.go

```diff
@@ -0,0 +1,73 @@
+package subcommands
+
+import (
+	"bytes"
+	"context"
+
+	pb "accretional.com/semantifly/proto/accretional.com/semantifly/proto"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+type Server struct {
+	pb.UnimplementedSemantiflyServer
+}
+
+const serverIndexPath = "index"
```

Comment:
Why is this a const? Shouldn't it be plumbed down from the command line arguments

Reply:
fixed